<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Customers</class>
 <widget class="QMainWindow" name="Form_Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiLight</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Информация о Заказчиках</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Изображения/Пользователи.png</normaloff>Изображения/Пользователи.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAddCustomer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить заказчика</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Добавить.png</normaloff>Изображения/Добавить.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Requisites">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_DataDocument">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Данные_Документа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_DataDocument">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_INN">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ИНН</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_INN">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_InfoaboutCustomers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация о Заказчиках</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ViewCustomer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вид_Контрагента</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Наименование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Requisites">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Банковские реквизиты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить поля</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Очистить.png</normaloff>Изображения/Очистить.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Correctly">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Редактирование таблицы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущая запись</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Назад.png</normaloff>Изображения/Назад.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующая запись</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Вперед.png</normaloff>Изображения/Вперед.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>На Главную</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Изображения/Выход.png</normaloff>Изображения/Выход.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_NumberCustomer">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_NumberCustomer">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>110</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>№_Контрагента</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>210</y>
      <width>293</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnGoCustomer">
       <property name="text">
        <string>Перейти к заказчику</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Перейти.png</normaloff>Изображения/Перейти.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCorrectly">
       <property name="text">
        <string>Редактировать данные</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Редактировать.png</normaloff>Изображения/Редактировать.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Удалить данные</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Изображения/Удалить.png</normaloff>Изображения/Удалить.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_login">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_password">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_ViewCustomer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Document">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Document">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Идентиф_Документ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_KPP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_KPP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>КПП</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
